Storage/src/SigningHelper.php

```diff
@@ -337,9 +343,15 @@ public function v4Sign(ConnectionInterface $connection, $expires, $resource, $ge
             $requestHash
         ]);
 
-        $signature = bin2hex(base64_decode($credentials->signBlob($stringToSign, [
-            'forceOpenssl' => $options['forceOpenssl']
-        ])));
+        $signature = bin2hex(base64_decode($this->retrySignBlob(function () use (
+            $credentials,
+            $stringToSign,
+            $options
+        ) {
+            return $credentials->signBlob($stringToSign, [
+                'forceOpenssl' => $options['forceOpenssl']
+            ]);
+        })));
```

Comment:
nit: may be cleaner to use arrow functions
```suggestion
        $signature = bin2hex(base64_decode($this->retrySignBlob(
            f() => $credentials->signBlob($stringToSign, [
                'forceOpenssl' => $options['forceOpenssl']
            ]);
        ));
```

Reply:
Thank you for your suggestion regarding the use of arrow functions. I have reviewed the code and applied the changes as recommended. It does indeed make the code cleaner and more concise.